src/Jwt/Symfony/Authentication/JwtAuthenticationProvider.php

```diff
@@ -51,9 +51,9 @@ public function authenticate(TokenInterface $token)
             );
         }
 
-        if (!$this->decoderService->validateData($jwt)) {
+        if (!$this->decoderService->validateTimeSensitiveClaims($jwt)) {
             throw new AuthenticationException(
-                'Token claims validation failed. This most likely means the token is expired.'
+                'Token expired (or not yet usable yet).'
```

Comment:
Nit: I think the last "yet" is not needed and "or not yet usable" is ok.

Reply:
Yeah good spot, it's a typo!